app/views/users/new.html.erb

```diff
@@ -1,3 +1,23 @@
 <% provide :title, t("static_pages.signup.title") %>
 <h1><%= t "static_pages.signup.title" %></h1>
-<p><%= t "static_pages.signup.content" %></p>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user, url: signup_path) do |f| %>
+      <%= render "shared/error_messages" %>
+
+      <%= f.label :name %>
```

Comment:
Những chỗ này đã set i18n chưa a?
Nếu chưa, a tìm hiểu `i18n for label` chứ gọi biến t(" ") như chỗ khác a nhé